src/components/sidebar/sidebar.stories.mdx

```diff
@@ -66,7 +66,7 @@ const MyComponent = ({ isOpen, handleClose }) => (
             <SidebarHeader>Header Content</SidebarHeader>
             <div>
               <Button as='primary'>Test</Button>
-              <Button as='secondary'>Last</Button>
+              <Button as='secondary' ml={2}>Last</Button>
```

Comment:
You need to make the same change for `sidebar.stories.js` as well please

Reply:
Done. Cheers for spotting that.